Misc/NEWS.d/next/Tests/2020-06-17-17-27-07.bpo-41009.Rvn6OQ.rst

```diff
@@ -0,0 +1,2 @@
+Fix use of ``support.require_{linux|mac|freebsd}_version()`` decorator as
+class decorator.
```

Comment:
Rather than "fix", I would say that it's an enhancement ;-) It becomes possible to use them on classes ;-)

decorator => decorators

Reply:
No, it's actually a fix. Decoratored classes are ignored and their tests are not executed at all.